webpack/assets/javascripts/react_app/components/Architectures/Welcome.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { translate as __ } from '../../common/I18n';
+import EmptyState from '../common/EmptyState';
+import { foremanUrl } from '../../common/helpers';
+
+export const WelcomeArchitecture = ({ canCreate }) => {
+  const action = canCreate && {
+    title: __('Create Architecture'),
+    url: foremanUrl('/architectures/new'),
+  };
+  const content = __(`Each entries represent a particular hardware architecture, most commonly <b>x86_64</b> or <b>i386</b>.
+  Foreman also supports the Solaris operating system family, which includes <b>sparc</b> based systems.`);
+  const description = (
+    <>
+      {__(
+        'Before you proceed to using Foreman you should provide information about one or more architectures.'
+      )}
+      <br />
```

Comment:
With the new `<div>` I think you don't need an extra `<br />` since the div would break the line anyway